mne/tests/test_epochs.py

```diff
@@ -905,7 +905,7 @@ def test_crop():
     assert_array_equal(data3, data_normal[:, :, tmask])
 
     # test time info is correct
-    epochs = EpochsArray(np.zeros((1, 1, 1000)), create_info(1, 1000., 'eeg'),
+    epochs = EpochsArray(np.zeros((1, 1, 1001)), create_info(1, 1000., 'eeg'),
```

Comment:
the test fails if you don't update this line?

or it's to test the fix?

if it's the latter then you should have both tests